iwyu_tool.py

```diff
@@ -362,6 +363,17 @@ def slice_compilation_db(compilation_db, selection):
 
         new_db.extend(found)
 
+    for path in exclude:
+        if not os.path.exists(path):
+            print('warning: \'%s\' not found on disk.' % path, file=sys.stderr)
```

Comment:
I wonder if this should say `warning: excluded [...]` to make it clearer that it's not the same warning as above?